pkg/media/media.go

```diff
@@ -9,12 +9,17 @@ import (
 
 // A Sample contains encoded media and timing information
 type Sample struct {
-	Data               []byte
-	Timestamp          time.Time
-	Duration           time.Duration
-	PacketTimestamp    uint32
-	PrevDroppedPackets uint16
-	Extensions         []rtp.Extension
+	Data                     []byte
+	Timestamp                time.Time
+	Duration                 time.Duration
+	PacketTimestamp          uint32
+	PrevDroppedPackets       uint16
+	Extensions               []rtp.Extension
+	Don                      uint16
```

Comment:
how about having all our attributes in a separate struct? 
```
type SampleHyperscaleExtensions struct {
	Don                      uint16
	IsIFrame                 bool
	IsSpsPps                 bool
	IsAbr                    bool
	WithHyperscaleExtensions bool
}

type Sample struct {
	Data                     []byte
	Timestamp                time.Time
	Duration                 time.Duration
	PacketTimestamp          uint32
	PrevDroppedPackets       uint16
	Extensions               []rtp.Extension
        SampleHyperscaleExtensions
}


```
that way we can easily know what is part of hyperscale and was is part of pion on the next merge..